hail/python/hailtop/hailctl/dev/benchmark/cli.py

```diff
@@ -23,14 +24,56 @@ def main(args_):
     parser.add_argument("--log", "-l",
                         type=str,
                         help='Log file path')
+    parser.add_argument("--verbose", "-v",
+                        action="store_true",
+                        help="Print testing information to stderr in real time.")
+    parser.add_argument("--output", "-o",
+                        type=str,
+                        help="Output file path.")
+    parser.add_argument("--format", "-f",
+                        type=str,
+                        help="Output format.",
+                        default='table',
+                        choices=['table', 'text'])
 
     args = parser.parse_args(args_)
 
-    initialize(args.cores, args.log, args.n_iter)
+    initialize(args)
+
+    finalizers = []
+    if args.output:
+        out_file = open(args.output, 'w')
+        finalizers.append(lambda: out_file.close())
+    else:
+        out_file = None
+    writer = lambda s: print(s, end='', file=out_file)
```

Comment:
What happens here if `out_file` is None? I would think you'd want `sys.stdout` by default, not None.

Reply:
Ok, just found in the docs that None is the same as stdout, so this is fine. Personally prefer explicitness of sys.stdout but no big deal.